model/project_ref.go

```diff
@@ -1062,6 +1062,38 @@ func CountProjectRefsWithIdentifier(identifier string) (int, error) {
 	return db.CountQ(ProjectRefCollection, byId(identifier))
 }
 
+// GetTasksWithOptions will find the last number of tasks (denoted by NumVersions) that have run on a given project.
+// This function may also filter on tasks running on a specific build variant, or tasks that come after a specific revision order number.
+func GetTasksWithOptions(projectName string, taskName string, opts GetProjectTasksOpts) ([]task.Task, error) {
```

Comment:
This doesn't specifically filter tasks that have _run_. A big reason cloud wants this is to filter out non-activated / scheduled tasks so that they don't have to paginate. We should also reflect this in the test.